tee-worker/cli/src/lib.rs

```diff
@@ -45,17 +46,26 @@ pub mod commands;
 
 use crate::commands::Commands;
 use clap::Parser;
+use itp_node_api::api_client::Metadata;
+use sp_application_crypto::KeyTypeId;
 use sp_core::{H160, H256};
-use substrate_api_client::Metadata;
 use thiserror::Error;
 
 const VERSION: &str = env!("CARGO_PKG_VERSION");
 
+pub(crate) const SR25519_KEY_TYPE: KeyTypeId = KeyTypeId(*b"sr25");
+pub(crate) const ED25519_KEY_TYPE: KeyTypeId = KeyTypeId(*b"ed25");
+
 #[derive(Parser)]
 #[clap(name = "litentry-cli")]
 #[clap(version = VERSION)]
-#[clap(author = "Trust Computing GmbH <info@litentry.com>")]
+#[clap(
+	author = "Trust Computing GmbH <info@litentry.com>, Integritee AG <hello@integritee.network>"
+)]
 #[clap(about = "cli tool to interact with litentry-parachain and workers", long_about = None)]
+#[cfg_attr(feature = "teeracle", clap(about = "interact with integritee-node and teeracle", long_about = None))]
+#[cfg_attr(feature = "sidechain", clap(about = "interact with integritee-node and sidechain", long_about = None))]
+#[cfg_attr(feature = "offchain-worker", clap(about = "interact with integritee-node and offchain-worker", long_about = None))]
```

Comment:
I think you need to change to litentry version

Reply:
done.